[Bug]: I can't launch version 2.00 of the game.

### 👉 Please confirm the following

- [x] I agree to follow the [Code of Conduct](https://github.com/dylan-power/dinosaur-exploder/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I checked the [current issues](https://github.com/dylan-power/dinosaur-exploder/issues) for duplicate bugs.

### ❌ Current Behavior

When I try to run the command:
```
 java -jar target/dinosaur-exploder-2.0.0.jar 
```
I get the message:
```
no main manifest attribute, in target/dinosaur-exploder-2.0.0.jar
```
Specifying the main class explicitly doesn't help:
```
 > java -cp target/dinosaur-exploder-2.0.0.jar com.dinosaur.dinosaurexploder.DinosaurApp 
Error: Could not find or load main class com.dinosaur.dinosaurexploder.DinosaurApp
Caused by: java.lang.NoClassDefFoundError: com/almasb/fxgl/app/GameApplication
```
However, there are no such problems with version 1.0:
```
java -jar target/dinosaur-exploder-1.0.jar 
```
java -cp target/dinosaur-exploder-1.0.jar com.dinosaur.dinosaurexploder.DinosaurApp 
```
Both methods launch the application.
I'm running it on Ubuntu 22.4.

### ✅ Expected Behavior

_No response_

### 🚩 Steps To Reproduce

_No response_

### 💻 Environment

```markdown
- OS:
- Game version: 1.2.0
```

### ℹ️ Additional information

_No response_

--- a/README.md
+++ b/README.md
@@ -120,6 +120,10 @@ cd dinosaur-exploder
 mvn clean install
 mvn javafx:run
 
+# Or build with desktop profile
+mvn clean package -Pdesktop
+java -jar target/dinosaur-exploder-2.0.0.jar
+
 # Or using an IDE: 
 # - Open project in IntelliJ
 # - Run com.dinosaur.dinosaurexploder.DinosaurApp or hit the run button at the top right

